app/charts/column/columns-state.tsx

```diff
@@ -102,7 +103,22 @@ const useColumnsState = ({
   const getX = useStringVariable(fields.x.componentIri);
   const getXAsDate = useTemporalVariable(fields.x.componentIri);
   const getY = useOptionalNumericVariable(fields.y.componentIri);
+  const errorIri = useMemo(() => {
+    const yMeasure = measures.find((m) => m.iri === fields.y.componentIri);
+    return yMeasure?.related?.errorIri;
+  }, [fields.y.componentIri, measures]);
   const getSegment = useSegment(fields.segment?.componentIri);
+  const getYError = errorIri
+    ? (d: Observation) => {
+        const y = getY(d) as number;
+        const error =
+          d[errorIri] !== null ? parseFloat(d[errorIri] as string) : null;
+        return (error === null ? [y, y] : [y - error, y + error]) as [
+          number,
+          number
+        ];
+      }
+    : null;
```

Comment:
I would expect to get just a yError when I call `getYError`, not yValue +- yError; maybe the function could be called `getYAdjustedByError`, or similarly?

Reply:
getYErrorRange ?

Good point about the scales, I will do it !

Reply:
I think `getYErrorRange` it's also ok :)

Reply:
OK thanks, I'll do that in the new PR.